diff --git a/sc/qa/extras/macros-test.cxx b/sc/qa/extras/macros-test.cxx
index 68bed14..4251f30 100644
--- a/sc/qa/extras/macros-test.cxx
+++ b/sc/qa/extras/macros-test.cxx
@@ -136,10 +136,6 @@ void ScMacrosTest::testVba()
             OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
         },
         {
-            OUString("Ranges-2."),
-            OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
-        },
-        {
             OUString("pagesetup."),
             OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
         },
@@ -160,10 +156,6 @@ void ScMacrosTest::testVba()
             OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
         },
         {
-            OUString("Ranges."),
-            OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
-        },
-        {
             OUString("CheckOptionToggleValue."),
             OUString("vnd.sun.Star.script:VBAProject.testMacros.test?language=Basic&location=document")
         },
