From 4ecd3608b6af165de5c8b9988943e924bd73f878 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Thu, 6 Nov 2014 11:35:35 +0100
Subject: [PATCH 1/2] debian/shlibs.override.icu:update to actual current
 SOVERSION...

---
 changelog           |    4 +++-
 shlibs.override.icu |   10 +++++-----
 2 files changed, 8 insertions(+), 6 deletions(-)

diff --git a/changelog b/changelog
index 87dddd1..5fc6389 100644
--- a/changelog
+++ b/changelog
@@ -6,8 +6,10 @@ libreoffice (1:4.4.0~beta1~git20141103-1) UNRELEASED; urgency=medium
     - add conditional/includes/libs for system-opencollada
     - use libmariadb-client-lgpl-dev instead of libmariadbclient-dev
       when USE_MARIADB=y
+  * debian/shlibs.override.icu:
+    - update to actual cuurent SOVERSION...
 
- -- Rene Engelhard <rene@debian.org>  Mon, 20 Oct 2014 18:46:46 +0200
+ -- Rene Engelhard <rene@debian.org>  Thu, 06 Nov 2014 11:25:45 +0100
 
 libreoffice (1:4.4.0~alpha1-3) experimental; urgency=low
 
diff --git a/shlibs.override.icu b/shlibs.override.icu
index b953094..82f557a 100644
--- a/shlibs.override.icu
+++ b/shlibs.override.icu
@@ -1,5 +1,5 @@
-libicudata	40
-libicui18n	40
-libicule	40
-libicuuc	40
-libicutu	40
+libicudata	53
+libicui18n	53
+libicule	53
+libicuuc	53
+libicutu	53
-- 
1.7.10.4

