From bc4037f1a58936df831017848057f44b9cc285db Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Sat, 26 Oct 2013 19:57:10 +0200
Subject: [PATCH] rc2 == final; release 1:4.1.3-1

---
 changelog |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/changelog b/changelog
index 4d5761a..81bdeb7 100644
--- a/changelog
+++ b/changelog
@@ -1,8 +1,8 @@
-libreoffice (1:4.1.3~rc2-1) UNRELEASED; urgency=low
+libreoffice (1:4.1.3-1) unstable; urgency=low
 
-  * new upstream release candidate
+  * new upstream release
 
- -- Rene Engelhard <rene@debian.org>  Wed, 23 Oct 2013 09:03:15 +0200
+ -- Rene Engelhard <rene@debian.org>  Sat, 26 Oct 2013 14:45:05 +0200
 
 libreoffice (1:4.1.2-2) unstable; urgency=low
 
-- 
1.7.10.4

