From 70ff9c176bae9dca5164b0bff1907f4ff28b9379 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Mon, 29 Jul 2013 15:40:54 +0200
Subject: [PATCH 4/5] move liblpsolve55.so into -core if used internally

---
 changelog |    2 ++
 rules     |    7 +++++++
 2 files changed, 9 insertions(+)

diff --git a/changelog b/changelog
index 357169f..1435e7c 100644
--- a/changelog
+++ b/changelog
@@ -2,6 +2,8 @@ libreoffice (1:4.1.0-3) unstable; urgency=low
 
   * debian/rules:
     - use system-clucene again now that it's in sid
+    - move liblpsolve55.so into -core if used internally, as otherwise
+      -writer doesn't start without -calc installed..?!
   * debian/libreoffice-common.postrm.in:
     - add missed || true and use rm -f for javasettinsgunopkginstall.xml...
   * debian/control.librelogo.in:
diff --git a/rules b/rules
index 6ed030d..f5073c6 100755
--- a/rules
+++ b/rules
@@ -2527,6 +2527,13 @@ else
 	rm -f $(PKGDIR)-core/$(OODIR)/program/ui-previewer
 endif
 
+# see https://bugs.freedesktop.org/show_bug.cgi?id=67349
+# writer doesn't start without it.
+ifeq (,$(filter lpsolve, $(SYSTEM_STUFF)))
+	mv $(PKGDIR)-calc/$(OODIR)/program/liblpsolve55.so \
+		$(PKGDIR)-core/$(OODIR)/program
+endif
+
 	for i in $(ARCH_DEP_PACKAGES); do \
 		if [ -e debian/$$i.bug-script.in ]; then \
 			cat debian/$$i.bug-script.in \
-- 
1.7.10.4

