From 7e09f2f6bf163601b61579c97940f13856f1154b Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Mon, 29 Jul 2013 15:48:00 +0200
Subject: [PATCH 5/5] fix shlibs:Depends for liblibreoffice-dev

---
 control.liblibreoffice.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/control.liblibreoffice.in b/control.liblibreoffice.in
index 52cd0be..fa861ad 100644
--- a/control.liblibreoffice.in
+++ b/control.liblibreoffice.in
@@ -10,7 +10,7 @@ Package: liblibreoffice-dev
 Section: libdevel
 Priority: extra
 Architecture: %OOO_ARCHS%
-Depends: liblibreofficeFIXME (= ${Source:Version}), ${misc:Depends}, ${shlibs:D
+Depends: liblibreofficeFIXME (= ${Source:Version}), ${misc:Depends}, ${shlibs:Depends}
 Description: foo
  bar
 
-- 
1.7.10.4

